### PR TITLE
file upload in webui without skeleton

### PR DESCRIPTION
**Pull request by haribhandari07**
## Description

Do not use skeleton files for ```webUIUpload``` to speed up CI

## Related Issue

Part of https://github.com/owncloud/QA/issues/622

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/webUIUpload/upload.feature [hunk 1]
```diff
@@ -27,12 +28,16 @@ Feature: File Upload
   @skipOnFIREFOX
   Scenario: conflict with a chunked file
     Given a file with the size of "30000000" bytes and the name "big-video.mp4" has been created locally
+    And user "user1" has uploaded file with content "some content" to "/lorem.txt"
```
Comment (skshetry) [inline tests/acceptance/features/webUIUpload/upload.feature]: I personally prefer following step than creating the file ourself.
```suggestion
    And user "user1" has uploaded file "filesForUpload/lorem.txt" to "/lorem.txt"
```

**Review by dpakach — COMMENTED** [review]
LGTM :+1:

**Review by phil-davis — APPROVED** [review]

**Comment by HariBhandari07** [comment]
Backport in #35536

**Comment by phil-davis** [comment]
Random drone networking error. I restarted the drone build.

**Pull request merged**